### DISCUSSION TITLE
Welcome to Xteam Discussions!

### DISCUSSION THREAD

**Discussion by xploitstech** [xploitstech/Xteam#2] [Announcements]
## 👋 Welcome! hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by TheETR** [comment]
Can u guys add IP tracer or something like that?

**Reply by xploitstech** [reply]
To be added soon

**Comment by cppcw** [comment]
I try to use it and it says:

"Please make sure you have the correct access rights
and the repository exists.
Xteam.sh: line 61: cd: zphisher: No such file or directory
bash: zphisher.sh: No such file or directory"
